docs/ENVIRONMENT.md

```diff
@@ -343,7 +343,7 @@ the journal instead of only when logging in debug mode.
 
 * `SOURCE_DATE_EPOCH` — if unset, the field of the date of last password change
```

Comment:
Add `$` here as well?

Reply:
Addressed in https://github.com/systemd/systemd/pull/27298.